### PR TITLE
Added Tag API

### PR DESCRIPTION
**Pull request by truechernyshov**

### CODE REVIEW

**Comment by alekseyp** [comment]
@ashkarpetin @2e3s @adamaltman  

sorry to bother you, but would love to see if this can be merged.

File: src/Entities/Tag.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        return $this->getAttribute('createdTime');
+    }
+
+    /**
+     * @return string
+     */
+    public function getUpdatedTime()
+    {
+        return $this->getAttribute('updatedTime');
+    }
```
Comment (ashkarpetin) [inline src/Entities/Tag.php:35]:
```suggestion
```

It's already defined in parent class

File: src/Services/TagService.php [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+     * @throws DataValidationException The input data does not valid
+     *
+     * @return Tag
+     */
+    public function create($data, $tag = null)
+    {
+        if (isset($tag)) {
+            return $this->client()->patch($data, 'tags/{tag}', ['tag' => $tag]);
+        }
+
```
Comment (ashkarpetin) [inline src/Services/TagService.php:114]:
```suggestion
```

This method is only for post/put

File: src/Services/TagService.php [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        return $this->client()->post($data, 'tags');
+    }
```
Comment (ashkarpetin) [inline src/Services/TagService.php:116]:
```suggestion
    /**
     * @param array|JsonSerializable|Tag $data
     *
     * @throws DataValidationException The input data does not valid
     *
     * @return Tag
     */
    public function create($data)
    {
        return $this->client()->post($data, 'tags');
    }
```

This method is only for post/put

**Review by ashkarpetin — APPROVED** [review]

**Review by 2e3s — APPROVED** [review]

**Comment by ashkarpetin** [comment]
Thanks!

**Pull request merged**